05_classify_mutations_multimodal/nbconverted/plot_multimodal_results.py

```diff
@@ -105,104 +112,67 @@
 
 
 # get results from unimodal prediction (individual data types) to compare with
-# these are spread across 3 different results directories
-# this is such a mess, I really need to clean up the results directories eventually
-
-# expression, seed 1
-unimodal_results_dir_1 = Path(
-    cfg.results_dirs['mutation'],
-    'bmiq_results',
-    'gene'
-)
-
-# expression, seed 2
-unimodal_results_dir_2 = Path(
-    cfg.results_dirs['mutation'],
-    'bmiq_results_2',
-    'gene'
-)
-
-# me_27k, both seeds
-unimodal_results_dir_3 = Path(
-    cfg.results_dirs['mutation'],
-    'bmiq_results_me_control',
-    'gene'
-)
-
-# me_450k, both seeds
-# use compressed data for this
-unimodal_results_dir_4 = Path(
-    cfg.results_dirs['mutation'],
-    'compressed_results',
-    'gene'
-)
-
-
-# In[6]:
+if not RAW_FEATS:
+    unimodal_results_dir = Path(
+        cfg.results_dirs['mutation'],
+        'methylation_results',
+        'gene'
+    )
 
+    # load expression and me_27k results
+    u_results_df = au.load_compressed_prediction_results(unimodal_results_dir, 'gene')
+    u_results_df = u_results_df[(u_results_df.n_dims == 5000)].copy()
+    u_results_df.drop(columns='n_dims', inplace=True)
 
-# load expression and me_27k results
-u_results_df_1 = au.load_stratified_prediction_results(unimodal_results_dir_1, 'gene')
-u_results_df_1 = u_results_df_1[(u_results_df_1.training_data == 'expression') &
-                                (u_results_df_1.seed == 1)].copy()
-u_results_df_2 = au.load_stratified_prediction_results(unimodal_results_dir_2, 'gene')
-u_results_df_2 = u_results_df_2[u_results_df_2.training_data == 'expression'].copy()
-u_results_df_3 = au.load_stratified_prediction_results(unimodal_results_dir_3, 'gene')
-
-# load me_450k results
-u_results_df_4 = au.load_compressed_prediction_results(unimodal_results_dir_4, 'gene',
-                                                       old_filenames=True)
-u_results_df_4 = u_results_df_4[(u_results_df_4.training_data == 'me_450k') &
-                                (u_results_df_4.n_dims == 5000)].copy()
-u_results_df_4.drop(columns='n_dims', inplace=True)
-
-# make sure data loaded matches our expectations
-print(u_results_df_1.training_data.unique())
-print(u_results_df_2.training_data.unique())
-print(u_results_df_3.training_data.unique())
-print(u_results_df_4.training_data.unique())
-print(u_results_df_1.seed.unique())
-print(u_results_df_2.seed.unique())
-print(u_results_df_3.seed.unique())
-print(u_results_df_4.seed.unique())
+    # make sure data loaded matches our expectations
+    print(u_results_df.training_data.unique())
+    print(u_results_df.seed.unique())
```

Comment:
You may want to make these assert statements instead of print statements if their goal is to make sure nothing is changing behind the scenes

Reply:
These are less to make sure _nothing_ is changing, and more just to visualize what's changing as I apply this script to different sets of results. The print statements allow me to see at a glance how many replicates/CV folds I ran (and make sure it's what I expected), as well as what models I ran (and make sure they're what I expected as well).

I guess since these results are more or less finalized for the paper now, I'm open to taking these out - let me know what you think.

Reply:
I think leaving them in or removing them is fine. The comment just gave me the impression that the statements were supposed to always return the same thing.